### PR TITLE
[Fix] Gogdl issues don't fail gracefully #1125

### PR DESCRIPTION
**Pull request by dawidgarus**
Fixes https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1125

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by flavioislima** [comment]
I think one has a small conflict now after the other merge.

File: src/screens/Library/components/InstallModal/index.tsx [hunk 1]
```diff
@@ -196,6 +205,7 @@ export default function InstallModal({
         )
       }
       const regexp = new RegExp(/[:|/|*|?|<|>|\\|&|{|}|%|$|@|`|!|™|+|']/, 'gi')
+      console.log(gameInfo, gameData)
```
Comment (flavioislima) [inline src/screens/Library/components/InstallModal/index.tsx:208]: Remember to remove before mergin 👍🏽

Comment (dawidgarus) [reply src/screens/Library/components/InstallModal/index.tsx]: I missed that one. Maybe we should add eslint rules to ban console.log?

Comment (flavioislima) [reply src/screens/Library/components/InstallModal/index.tsx]: yes, we should 👍🏽

**Review by flavioislima — APPROVED** [review]
LGTM! 💯

**Pull request merged**